blockchain/api.go

```diff
@@ -185,6 +188,13 @@ type MultiSigAPI interface {
 	GetTransaction(ctx context.Context, transactionID *big.Int) (*MultiSigTransactionData, error)
 }
 
+type DeviceStorageAPI interface {
+	StoreOrUpdate(ctx context.Context, key *ecdsa.PrivateKey, devices interface{}) error
+	Devices(ctx context.Context, address common.Address) (*sonm.StoredDevicesReply, error)
+	// For debug purposes, when something could not be properly unmarshalled for example
+	RawDevices(ctx context.Context, address common.Address) ([]byte, uint64, error)
```

Comment:
Since we already return `sonm.StoredDevicesReply` in the method above, why can't we do the same thing here returning `sonm.RawDevicesReply` instead?